test/amp-caching/amp-caching-test.js

```diff
@@ -0,0 +1,79 @@
+describe('Amp caching', function() {
+  let driver;
+
+  before(async () => {
+    driver = global.__AMPSW.driver;
+    await driver.get('http://localhost:6881/test/index.html');
+  });
+
+  beforeEach(async () => {
+    await driver.navigate().refresh();
+    await driver.executeAsyncScript(async cb => {
+      await window.__testCleanup();
+      const registration = await navigator.serviceWorker.register('/test/amp-caching-sw.js');
+      await window.__waitForSWState(registration, 'activated');
+      cb();
+    });
+    await driver.navigate().refresh();
+    const swRegCount = await driver.executeAsyncScript(async cb => {
+      const regs = await navigator.serviceWorker.getRegistrations();
+      cb(regs.length);
+    });
+    expect(swRegCount).to.be.equal(1);
+  });
+
+  afterEach(async () => {
+    await driver.executeAsyncScript(async cb => {
+      await window.__testCleanup();
+      cb();
+    });
+  });
+
+  describe('Versioned JS', () => {
+    const versionedAmpRuntime = 'https://cdn.ampproject.org/rtv/001525381599226/v0.js';
+    //const versionedAmpExtension = 'https://cdn.ampproject.org/rtv/001525381599226/v0.js';
+
+    ['https://cdn.ampproject.org/rtv/001525381599226/v0.js', 'https://cdn.ampproject.org/rtv/001525381599226/v0.js'].forEach(() => {
```

Comment:
I'm confused, an array literal of the same two values as the const?

Reply:
fixed to use all kinds of files that amp deploys